Modules/System/AL Rest Client/src/ALHttpContent.Codeunit.al

```diff
@@ -0,0 +1,261 @@
+/// <summary>Holder object for the Http Content data.</summary>
+codeunit 2354 "AL Http Content"
+{
```

Comment:
We have a convention of using the Facade pattern for system application modules. Meaning that public codeunits should not have any logic code present. In some cases it may just seem like extra work for very little readability gain, but it does make it easier to navigate the application when consistently applied.

Reply:
Applied the Facade pattern to these codeunits:
* Http Content
* Http Request Message
* Http Response Message